libs/sdk-ui-kit/src/@ui/UiMenu/components/defaults/DefaultUiMenuGroupItem.tsx

```diff
@@ -28,6 +28,9 @@ export function DefaultUiMenuGroupItemComponent<T extends IUiMenuItemData = obje
                 <ShortenedText className={e("group-title")} ellipsisPosition={"end"}>
                     {item.stringTitle}
                 </ShortenedText>
+                <div className={e("group-separator")} role={"presentation"}>
+                    <SeparatorLine mL={10} />
```

Comment:
I know it's not your code but these mL, rL, etc. properties are horrible.

Reply:
expect this idea comes from some library. Btw separator is not part of final design, so removed from default component